CodeBeam.MudExtensions/Components/Loading/MudLoading.razor.cs

```diff
@@ -64,6 +64,27 @@ public bool Loading
         [Category(CategoryTypes.FormComponent.Appearance)]
         public string ClassText { get; set; }
 
+        /// <summary>
+        /// CSS classes for the progress component, seperated by space.
+        /// </summary>
+        [Parameter]
+        [Category(CategoryTypes.FormComponent.Appearance)]
+        public string ClassLoader { get; set; }
+
+        /// <summary>
+        /// CSS style for the text.
+        /// </summary>
+        [Parameter]
+        [Category(CategoryTypes.FormComponent.Appearance)]
+        public string TextStyle { get; set; }
```

Comment:
Sorry for another thing again, we should name `Class` and `Style` first like `StyleText`. So when user write Style, they show all parameters that about Style.

Reply:
fixed